pymc3/variational/advi.py

```diff
@@ -324,17 +324,17 @@ def rvs(x):
     for v in global_RVs:
         u = theano.shared(vparams['means'][str(v)]).ravel()
         w = theano.shared(vparams['stds'][str(v)]).ravel()
-        n = r.normal(size=u.tag.test_value.shape)
-        updates.update({v: (n * w + u).reshape(v.tag.test_value.shape)})
+        n = r.normal(size=np.shape(u.tag.test_value))
+        updates.update({v: (n * w + u).reshape(np.shape(v.tag.test_value))})
 
     if local_RVs is not None:
         for v_, (uw, _) in local_RVs.items():
             v = get_transformed(v_)
             u = uw[0].ravel()
             w = uw[1].ravel()
-            n = r.normal(size=u.tag.test_value.shape)
+            n = r.normal(size=np.shape(u.tag.test_value))
             updates.update(
-                {v: (n * tt.exp(w) + u).reshape(v.tag.test_value.shape)})
+                {v: (n * tt.exp(w) + u).reshape(np.shape(v.tag.test_value))})
```

Comment:
`test_value` was returning a primitive (not a numpy array) in the tests.  Not sure if that's due to another issue, yet; still clearing out the big issues following the evaluation hack I just added.  Will comment on all this once the tests are passing again (or sooner, if necessary).